wikimedia/wikipedia-ios
# Track notifications interactions

**Phabricator:** 

### Notes
* This PR implements the analytics tracking for the notifications. We are using the new analytics service to track the new iOS Notifications schema.

### Test Steps
1. Launch the app, and log into your account to access your notifications.
2. Click on the notification, and from the detail view, click on some of the actions.
3. From the swipe action, choose either mark as read/unread or one of the `More` actions from the action sheet
4. Select mark as read/unread from the Edit option
5. You should be able to check (with a network inspector) the calls to our API, be sure they're successful, and are sending the correct data for each action
